phpcs.xml

```diff
@@ -0,0 +1,39 @@
+<ruleset name="coding-style">
```

Comment:
is this  still necessary? @jvegalo you already created a CI job, but I don't know if this settings from this file are still correct

Reply:
According to the creator of this GIT Action, the phpcs.xml is necessary, you can see it in the last paragraph here: https://github.com/marketplace/actions/action-php-codesniffer

If I am not understanding the question, please let me know.

Reply:
I was asking since there was already a working CI action. If the file is needed in this PR then all good